components/ServiceList/index.js

```diff
@@ -0,0 +1,42 @@
+import useSWR from "swr";
+import Image from "next/image";
+
+export default function ServiceList() {
+  const { data, isLoading } = useSWR("/api/services");
+
+  if (isLoading) {
+    return <h2>Loading...</h2>;
+  }
+  console.log(data);
```

Comment:
**Please remove `console.log()` statements 🧹**

Leaving `console.log()` statements in a production app can be problematic for several reasons:

- Seeing `console.log()` messages displayed in a browser's console can give the impression of unprofessionalism, especially if these messages make no sense to the end user.
- Can expose sensitive information about how your application works, business logic, APIs used, or even user data. 

```suggestion

```